### PR TITLE
Add information about Text indexes and dictionary variables

### PR DESCRIPTION
**Pull request by JPryce-Aklundh**

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-cypher-1026.surge.sh

Updated pages:
- [The impact of indexes on query performance](https://neo4j-docs-cypher-1026.surge.sh/cypher-manual/5/indexes/search-performance-indexes/using-indexes)

File: modules/ROOT/pages/indexes/search-performance-indexes/using-indexes.adoc [hunk 1]
```diff
@@ -243,6 +243,28 @@ Text indexes are only used for exact query matches. To perform approximate match
 
 For more information about the predicates supported by text indexes, see xref:indexes/search-performance-indexes/managing-indexes.adoc#text-indexes-supported-predicates[Create, show, and delete indexes -> Text indexes: supported predicates].
 
+[[text-index-dictionary-variables]]
+=== Text indexes and dictionary variables
```
Comment (inmost-light) [inline modules/ROOT/pages/indexes/search-performance-indexes/using-indexes.adoc]: "Dictionary" and "map" are often used interchangeable, in Cypher we call them "maps" - https://neo4j.com/docs/cypher-manual/current/values-and-types/maps/

File: modules/ROOT/pages/indexes/search-performance-indexes/using-indexes.adoc [hunk 2]
```diff
@@ -243,6 +243,28 @@ Text indexes are only used for exact query matches. To perform approximate match
...
+
+Text indexes are not used to solve predicates with dictionary variables, such as JSON.
```
Comment (inmost-light) [inline modules/ROOT/pages/indexes/search-performance-indexes/using-indexes.adoc]: Not sure about "such as JSON" wording. You can use Cypher maps to represent JSON objects...

**Review by inmost-light — CHANGES_REQUESTED** [review]

**Review by inmost-light — APPROVED** [review]

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-cypher-1026.surge.sh

Updated pages:
- [The impact of indexes on query performance](https://neo4j-docs-cypher-1026.surge.sh/cypher-manual/5/indexes/search-performance-indexes/using-indexes)

**Comment by neo-technology-commit-status-publisher** [comment]
Thanks for the documentation updates.

The preview documentation has now been torn down - reopening this PR will republish it.

**Pull request merged**